README.md

```diff
@@ -21,7 +21,7 @@ The following workflows are supported by this repository:
 * Dissemination - https://github.com/sul-dlss/workflow-server-rails/blob/main/config/workflows/disseminationWF.xml
   * cleans up the workspace after accessioning
 * Release - https://github.com/sul-dlss/workflow-server-rails/blob/main/config/workflows/releaseWF.xml
-  * Moves files to PURL and updates the marc record in the ILS
+  * Moves files to PURL and updates the marc record in the ILS catalog (for SearchWorks consumption)
```

Comment:
For those of us that didn't have the joy of spending the first part of their DLSS tenure working on SW, "for SearchWorks consumption" is actually not that clear. How about "adding fields that are needed for SearchWorks indexing and display"?